integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -1098,9 +1096,65 @@ public void testConfiguredServerLifeCycleWithoutAdmin() {
     }
   }
 
-  private static void createDomainSecret(String secretName, String username, String password, String domNamespace)
-          throws ApiException {
-    Map<String, String> secretMap = new HashMap();
+  /**
+   * Restart the clustered managed server that is part of the dynamic cluster using the sample scripts
+   * stopServer.sh and startServer.sh while keeping the replica count constant.
+   * The test case verifies the fix for OWLS-87209 where scripts don't work when serverStartState
+   * is set at the managed server level but serverStartPolicy is not set at the managed server level.
+   */
+  @Order(17)
+  @Test
+  @DisplayName("Restart the dynamic cluster managed server using sample scripts")
+  public void testManagedRestartWithConstantReplicaCount() {
+    String serverName = managedServerNamePrefix + 1;
+    String serverPodName = managedServerPrefix + 1;
+    String keepReplicasConstant = "-k";
+
+    // shut down the dynamic cluster managed server using the script stopServer.sh and keep replicas constant
+    executeLifecycleScript(STOP_SERVER_SCRIPT, SERVER_LIFECYCLE, serverName, keepReplicasConstant);
+    checkPodDeleted(serverPodName, domainUid, domainNamespace);
+    logger.info("managed server " + serverName + " stopped successfully.");
+
+    // start the dynamic cluster managed server using the script startServer.sh and keep replicas constant
+    executeLifecycleScript(START_SERVER_SCRIPT, SERVER_LIFECYCLE, serverName, keepReplicasConstant);
+    checkPodReadyAndServiceExists(serverPodName, domainUid, domainNamespace);
+    logger.info("managed server " + serverName + " restarted successfully.");
+  }
+
+  /**
+   * Restart the clustered managed server that is part of the dynamic cluster using the sample scripts
+   * stopServer.sh and startServer.sh along with changing the replica count.
+   * The test case verifies the fix for OWLS-87209 where scripts don't work when serverStartState
+   * is set at the managed server level but serverStartPolicy is not set at the managed server level.
+   */
+  @Order(18)
+  @Test
+  @DisplayName("Restart the dynamic cluster managed server using sample scripts")
```

Comment:
Can you make the DisplayName string unique for both tests #17 and  #18. 
Modify the test name to  testManagedRestartWithDynamicReplicaCount()--> testManagedWithExpilcitServerStartStateRestartWithDynamicReplicaCount